packages/components/src/components/CollapsibleBox/CollapsibleBox.stories.tsx

```diff
@@ -1,44 +1,31 @@
 import React from 'react';
 import styled from 'styled-components';
-import { storiesOf } from '@storybook/react';
-import { boolean, select } from '@storybook/addon-knobs';
+import { StoryObj } from '@storybook/react';
 
-import { CollapsibleBox } from './CollapsibleBox';
+import { CollapsibleBox as CollapsibleBoxComponent } from './CollapsibleBox';
 
 const Content = styled.div`
     width: 400px;
 `;
 
-storiesOf('Misc/CollapsibleBox', module).add('CollapsibleBox', () => {
-    const placement = select(
-        'Variant',
-        {
-            Large: 'large',
-            Small: 'small',
-            Tiny: 'tiny',
-        },
-        'small',
-    );
+export default {
+    title: 'Misc/CollapsibleBox',
+    component: CollapsibleBoxComponent,
+};
 
-    const headerJustifyContent = select(
-        'Header content alignent',
-        {
-            'Space between': 'space-between',
-            Center: 'center',
+export const CollapsibleBox: StoryObj = {
+    args: {
+        heading: 'Heading',
+        children: <Content>Some content</Content>,
+    },
+    argTypes: {
```

Comment:
`iconLabel` is missing an argType.

Reply:
08457ddc6bee6653a4c858ed6f3fb29cfdab558e